tests/raw_query/tests.py

```diff
@@ -307,3 +308,14 @@ def test_db_column_name_is_used_in_raw_query(self):
         """
         b = BookFkAsPk.objects.create(book=self.b1)
         self.assertEqual(list(BookFkAsPk.objects.raw('SELECT not_the_default FROM raw_query_bookfkaspk')), [b])
+
+    def test_DecimalField_filter(self):
+        """
+        Allow filtering by DecimalField in RawQuery.
+        Solve ticket #26219.
+        """
```

Comment:
You can remove this comment altogether as this test doesn't allow or solve anything.